src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsMetricAttributeGenerator.cs

```diff
@@ -366,6 +368,12 @@ private static string NormalizeRemoteServiceName(Activity span, string serviceNa
                 case "AmazonSQS": // AWS SDK v1
                 case "Sqs": // AWS SDK v2
                     return NormalizedSQSServiceName;
+                case "Bedrock":
+                case "Bedrock Agent":
+                case "Bedrock Agent Runtime":
+                    return NormalizedBedrockServiceName;
+                case "Bedrock Runtime":
+                    return NormalizedBedrockRuntimeServiceName;
```

Comment:
where are you using this ?

Reply:
NormalizeRemoteServiceName is used to determine the remote service from the upstream `aws.service` attribute as per [Cloud Control API](https://docs.aws.amazon.com/cloudcontrolapi/latest/userguide/supported-resources.html). Operations from BedrockAgent and BedrockAgentRuntime services are treated as AWS::Bedrock by the Cloud Formation template.

Reply:
There is no reference of `NormalizedBedrockRuntimeServiceName` in source code. If it is not used, please remove it from this PR.

Reply:
Ok. It is used to normalize "Bedrock Runtime".